alerter/src/main/java/com/tapadoo/alerter/Alert.java

```diff
@@ -176,7 +178,7 @@ public void onAnimationStart(final Animation animation) {
     @Override
     public void onAnimationEnd(final Animation animation) {
         //Start the Icon Animation once the Alert is settled
-        if (enableIconPulse) {
+        if (enableIconPulse && showIcon) {
```

Comment:
Update here to check the visibility of the icon, instead of using a boolean flag. 

`
if (enableIconPulse && ivIcon.getVisibility() == VISIBLE) {}
`